### DISCUSSION TITLE
Allowing submitting form by disabling enter keydown

### DISCUSSION THREAD

**Discussion by gizzmojo** [joshhanley/livewire-autocomplete#18] [Ideas]
I woud like to be remove disable this line without overwriting the whole file:
```html
x-on:keydown.enter.stop.prevent="enter($dispatch); event.target.blur()"
```

E.g
```html
<form action="/search">
    <x-lwa::autocomplete name="search" />
    <button type="submit">Search</button>
</form>
```

**Comment by joshhanley** [comment]
@gizzmojo currently, if an option is highlighted in the dropdown, then hitting enter selects that option, which is why that line exists. So I don't think deleting it would be good.

But I understand the need to be able to search for something and submit the search form if no result is selected.

I'd be open to a PR that adds that functionality 🙂

**Reply by howdu** [reply]
Thanks yeah that makes sence. Please see my PR: https://github.com/joshhanley/livewire-autocomplete/pull/19

I found the cleanest way was to conditionally add the 'preventDefault' than trying to manually submit the form. But maybe there's a better way.

**Reply by joshhanley** [reply]
Thanks! Will continue the discussion on the PR 🙂
